graphql_api/mutation/misc.py

```diff
@@ -51,8 +51,17 @@ def mutate(self, info, event_name: str, event_data: Dict[str, Any]):
             setattr(event_instance, key, event_data[key])
         doc_ock.post_event(str(user.id), event_instance)
         return UpdateConfig(ok=True)
+    
+class UpdateVersionsToNewest(graphene.Mutation):
+    ok = graphene.Boolean()
+
+    def mutate(self, info):
+        auth.check_demo_access(info)
+        manager.update_versions_to_newest()
```

Comment:
i think we should also check admin access so no everyone could in theory do this 
`auth.check_admin_access(info)`